src/filebrowser_upload/main.py

```diff
@@ -1,18 +1,23 @@
 # coding: utf-8
+
+import posixpath
 import argparse
 import sys
 import warnings
-from os.path import abspath, join, dirname, expanduser
-
-VENDOR_PATH = abspath(join(dirname(__file__), 'vendor'))
-sys.path.insert(0, VENDOR_PATH)
+from os import walk
+from os.path import abspath, dirname, expanduser, join
+from getpass import getpass
 
 import requests
 from tqdm import tqdm
 from urllib3.exceptions import InsecureRequestWarning
+
 from .__version__ import __version__
 
-warnings.simplefilter('ignore', InsecureRequestWarning)
+VENDOR_PATH = abspath(join(dirname(__file__), "vendor"))
```

Comment:
the code should run before import vender libs.

Reply:
Restored

Reply:
Reverted.. For me it is not working. It will trigger this error when using dist/filebrowser-upload
```
Traceback (most recent call last):
  File "<frozen zipimport>", line 177, in get_data
KeyError: 'certifi/cacert.pem'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen zipimport>", line 741, in open_resource
  File "<frozen zipimport>", line 179, in get_data
OSError: [Errno 0] : 'certifi/cacert.pem'
```